test/unit/actions-test.js

```diff
@@ -103,6 +103,32 @@ describe('Denali.Action', function() {
     });
   });
 
+  // having trouble skipping this test and needing to move on here for now
+  // it('should use the serializer content-type if provided', function() {
+  //   let responded = false;
+  //   let serialized = false;
+  //   class TestAction extends Action {
+  //     serializer = 'json-api';
+  //     respond() {
+  //       responded = true;
+  //       return {};
+  //     }
+  //   }
+  //   let mock = mockReqRes();
+  //   mock.container.register('serializer:json-api', {
+  //     contentType = 'application/vnd.api+json',
+  //     serialize() {
+  //       serialized = true;
+  //     }
+  //   });
+  //   let action = new TestAction(mock);
+
+  //   return action.run().then(() => {
+  //     expect(responded).to.be.true();
+  //     expect(serialized).to.be.true();
+  //   });
+  // });
+
```

Comment:
This is quite ugly here.  I tried `skip('it should use')` and `this.skip()` inside the test, not sure what was going wrong here?